docs/specification.md

```diff
@@ -0,0 +1,155 @@
+# Homebase Lite Spec Draft
+
+This spec draft is for the proposed Tezos smart contract to be used as the on-chain component of a new lightweight Homebase tool.
+
+# Requirements
+
+This is intended to be a simpler option than baseDAO, both in complexity and usage, for communities to request feedback, run polls, etc.
+
+In summary, the smart contract should support:
```

Comment:
Related to a previous comment: now that the draft has been approved we need to make this more assertive, given that this document now describes the goal (and eventually, the result).

So, here I'd change this:
```suggestion
In summary, the smart contract must support:

```